lib/reply.js

```diff
@@ -79,19 +77,18 @@ Object.defineProperties(Reply.prototype, {
   sent: {
     enumerable: true,
     get () {
-      return this[kReplySent]
+      return (this[kReplySentOverwritten] || (typeof this.raw.writableEnded !== 'undefined' ? this.raw.writableEnded : this.raw.finished)) === true
```

Comment:
Can a comment be added here to explain the inspections being performed?

Reply:
Done.